### PR TITLE
Added new Help Tab

### PR DESCRIPTION
**Pull request by reish981**

### CODE REVIEW

**Comment by kawesome** [comment]
Can one of the admins verify this patch?

File: src/main/java/edu/ksu/canvas/attendance/controller/AttendanceHelpController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@Scope("session")
+@RequestMapping("/help")
+public class AttendanceHelpController extends AttendanceBaseController{
+
+    private static final Logger LOG = Logger.getLogger(AttendanceHelpController.class);
```
Comment (KSU-Github) [inline src/main/java/edu/ksu/canvas/attendance/controller/AttendanceHelpController.java]: MAJOR Remove this unused "LOG" private field. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)

File: src/main/webapp/WEB-INF/jsp/help.jsp [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        <div class="panel-heading" role="tab" id="headingFour">
+            <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
+                    Understanding <em>Attendance Weights</em>
+                </a>
+            </h4>
+        </div>
+        <div id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour">
+            <div class="panel-body">
+                <img src="${context}/img/WeightsExample.png">
```
Comment (KSU-Github) [inline src/main/webapp/WEB-INF/jsp/help.jsp]: MINOR Add an "alt" attribute to this image. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/webapp/WEB-INF/jsp/help.jsp [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo"
+                   aria-expanded="false" aria-controls="collapseTwo">
+                    Pushing Grades
+                </a>
+            </h4>
+        </div>
+        <div id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
+            <div class="panel-body">
+                <img src="${context}/img/PushModal.png" border="5">
```
Comment (KSU-Github) [inline src/main/webapp/WEB-INF/jsp/help.jsp]: MAJOR Remove this deprecated "border" attribute. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)
MINOR Add an "alt" attribute to this image. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/webapp/WEB-INF/jsp/help.jsp [hunk 4]
```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<%@ taglib uri="http://www.springframework.org/tags/form" prefix="form" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
```
Comment (KSU-Github) [inline src/main/webapp/WEB-INF/jsp/help.jsp]: MAJOR Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)

File: src/main/webapp/WEB-INF/jsp/help.jsp [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+            <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne"
+                   aria-expanded="false" aria-controls="collapseOne">
+                Saving a Setup
+                </a>
+            </h4>
+        </div>
+        <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+            <div class="panel-body">
+                <img src="${context}/img/Setup.png" border="5">
```
Comment (KSU-Github) [inline src/main/webapp/WEB-INF/jsp/help.jsp]: MAJOR Remove this deprecated "border" attribute. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)
MINOR Add an "alt" attribute to this image. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

**Comment by KSU-Github** [comment]
SonarQube analysis reported 15 issues
* ![MAJOR][MAJOR] 8 major
* ![MINOR][MINOR] 7 minor

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [courseConfiguration.jsp#L8](https://github.com/kstateome/lti-attendance/blob/356acd63bcb6fbee87ba84a17e17cca6e283211f/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MAJOR][MAJOR] [roster.jsp#L9](https://github.com/kstateome/lti-attendance/blob/356acd63bcb6fbee87ba84a17e17cca6e283211f/src/main/webapp/WEB-INF/jsp/roster.jsp#L9): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MAJOR][MAJOR] [roster.jsp#L126](https://github.com/kstateome/lti-attendance/blob/356acd63bcb6fbee87ba84a17e17cca6e283211f/src/main/webapp/WEB-INF/jsp/roster.jsp#L126): Remove this deprecated "name" attribute. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)
1. ![MAJOR][MAJOR] [simpleAttendanceSummary.jsp#L7](https://github.com/kstateome/lti-attendance/blob/356acd63bcb6fbee87ba84a17e17cca6e283211f/src/main/webapp/WEB-INF/jsp/simpleAttendanceSummary.jsp#L7): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MINOR][MINOR] [courseConfiguration.jsp#L109](https://github.com/kstateome/lti-attendance/blob/356acd63bcb6fbee87ba84a17e17cca6e283211f/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L109): Add a <legend> tag to this fieldset. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)
1. ![MINOR][MINOR] [courseConfiguration.jsp#L141](https://github.com/kstateome/lti-attendance/blob/356acd63bcb6fbee87ba84a17e17cca6e283211f/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L141): Replace this <i> tag by <em>. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)
1. ![MINOR][MINOR] [courseConfiguration.jsp#L146](https://github.com/kstateome/lti-attendance/blob/356acd63bcb6fbee87ba84a17e17cca6e283211f/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L146): Replace this <i> tag by <em>. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)
1. ![MINOR][MINOR] [courseConfiguration.jsp#L152](https://github.com/kstateome/lti-attendance/blob/356acd63bcb6fbee87ba84a17e17cca6e283211f/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L152): Replace this <i> tag by <em>. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by reish981** [comment]
I know I'm probably missing some integration tests, but those will be added with the completion of LK1682.

File: src/main/java/edu/ksu/canvas/attendance/controller/AttendanceHelpController.java (hunk 1 above)
Comment (zoglmannk) [inline src/main/java/edu/ksu/canvas/attendance/controller/AttendanceHelpController.java]: I concur with this... please remove the unused logger

**Comment by KSU-Github** [comment]
SonarQube analysis reported 15 issues
* ![MAJOR][MAJOR] 8 major
* ![MINOR][MINOR] 7 minor

Watch the comments in this conversation to review them.

#### 9 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [attendanceSummary.jsp#L7](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/attendanceSummary.jsp#L7): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MAJOR][MAJOR] [courseConfiguration.jsp#L8](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MAJOR][MAJOR] [roster.jsp#L9](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/roster.jsp#L9): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MAJOR][MAJOR] [roster.jsp#L126](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/roster.jsp#L126): Remove this deprecated "name" attribute. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)
1. ![MAJOR][MAJOR] [simpleAttendanceSummary.jsp#L7](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/simpleAttendanceSummary.jsp#L7): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MINOR][MINOR] [courseConfiguration.jsp#L109](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L109): Add a <legend> tag to this fieldset. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)
1. ![MINOR][MINOR] [courseConfiguration.jsp#L141](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L141): Replace this <i> tag by <em>. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)
1. ![MINOR][MINOR] [courseConfiguration.jsp#L146](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L146): Replace this <i> tag by <em>. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)
1. ![MINOR][MINOR] [courseConfiguration.jsp#L152](https://github.com/kstateome/lti-attendance/blob/3926f29d14bc3fe5fc002ad06a3617be60057800/src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp#L152): Replace this <i> tag by <em>. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**